teachertool/src/components/SplitPane.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+// eslint-disable-next-line import/no-internal-modules
+import css from "./styling/SplitPane.module.css";
+import { classList } from "react-common/components/util";
+
+interface IProps {
+    className?: string;
+    split: "horizontal" | "vertical";
+    defaultSize: number | string;
+    primary: "left" | "right";
+    children: React.ReactNode;
+}
+
+const VerticalSplit: React.FC<IProps> = ({ className, split, children }) => {
```

Comment:
I want to switch away from the default export pattern for react components. Makes it too easy for actual component names to go stale, like this one. Something for a future PR.

Reply:
I'm a little confused about this comment because all of the components in this PR are using the `export default ..` pattern at the bottom of the file. Were we using the `export default` in the component declaration and you're wanting to use the `export default` pattern separate from the component declaration?

Reply:
I think we shouldn't `export default` components at all, so that we have to import them by their given name. It's the difference between this:
```ts
import SplitPane from "./SplitPane";
```
and this:
```ts
import { SplitPane } from "./SplitPane";
```

I'm suggesting we go with the second option, because it will enforce component name consistency. With the first option, this is equally valid:
```
import Button from "./SplitPane";
```
The result is you've imported SplitPane and called it Button. We should close off this vector for confusion.